ironfish/src/utils/currency.ts

```diff
@@ -6,10 +6,11 @@ const ORE_TICKER = '$ORE'
 const IRON_TICKER = '$IRON'
 const ORE_TO_IRON = 100000000
 export const MINIMUM_IRON_AMOUNT = 1 / ORE_TO_IRON
+export const MAXIMUM_IRON_AMOUNT = 1e12
```

Comment:
Do we need a maximum? We store most amounts in 64 bit ints in the rust, so technically the max is `1.8446744e+19`, I don't see a reason we need to enforce a max anywhere. I don't even see a reason we need to enforce a minimum either, but I've been meaning to remove the minimum too.

Reply:
i updated the max value. as to our conversation on discord, I don't know if you still want to enforce this logic to resolve https://github.com/iron-fish/ironfish/issues/699 or remove this altogether.